script/test

```diff
@@ -402,6 +402,14 @@ do_untar()
     rm "${build_dir:?}/${target_name}.tar"
 }
 
+do_copy_so_lib()
+{
+    mkdir so-lib
+    cp /lib/x86_64-linux-gnu/libgcc_s.so.1 ./so-lib
+    cp /lib/x86_64-linux-gnu/libc.so.6 ./so-lib
+    cp /lib64/ld-linux-x86-64.so.2 ./so-lib
```

Comment:
why is copying .so files necessary?

Reply:
In short, as the core dumps are generated on github-actions machines, we need the exact version of the main executable and shared libraries so that gdb backtrace can work. FYI: [https://lists.gnu.org/archive/html/help-gplusplus/2008-02/msg00031.html](https://lists.gnu.org/archive/html/help-gplusplus/2008-02/msg00031.html) 

I also provided the steps to analyze the core dumps: https://github.com/openthread/openthread/pull/5812#issuecomment-732749457

Reply:
Can we use the same ubuntu docker image to analyze the coredump so that we don't need upload Linux libraries like libc.so?

Reply:
I tried the approach using a ubuntu 18.04 docker image. However when using gdb in the container, backtrace still cannot dump the code correctly (showing `??`). Currently I didn't find any ways to get the exact same environment on github-actions to local. The *.so libs are really small in size, compared to binaries.

Reply:
Any actual size data?

Reply:
![Screenshot from 2020-11-25 14-52-47](https://user-images.githubusercontent.com/12945188/100193010-f1b2e900-2f2d-11eb-82cb-c6720f290bc9.png)


These are the artifacts downloaded from my test PR #5861. We can see the shared libs are **2.3M** in total.

Reply:
Sgtm :+1: